setup.py

```diff
@@ -178,9 +178,7 @@ def check_for_openmp():
         "url": 'https://khmer.readthedocs.org/',
         "packages": ['khmer', 'khmer.tests', 'oxli'],
         "package_dir": {'khmer.tests': 'tests'},
-        "install_requires": ['screed >= 0.9'],
-        # testing screed download link
-
+        "install_requires": ['screed >= 0.9', 'bz2file'],
```

Comment:
`bz2file` is already a requirement of screed

On Fri, Jul 31, 2015 at 7:23 AM C. Titus Brown <notifications@github.com>
wrote:

> In setup.py
> <https://github.com/dib-lab/khmer/pull/747#discussion_r35977820>:
>
> > @@ -178,9 +178,7 @@ def check_for_openmp():
> >          "url": 'https://khmer.readthedocs.org/',
> >          "packages": ['khmer', 'khmer.tests', 'oxli'],
> >          "package_dir": {'khmer.tests': 'tests'},
> > -        "install_requires": ['screed >= 0.9'],
> > -        # testing screed download link
> > -
> > +        "install_requires": ['screed >= 0.9', 'bz2file'],
>
> what happens if bz2file isn't available? I'm -0 on making this a hard
> requirement (although it may come with all the Pythons now).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dib-lab/khmer/pull/747/files#r35977820>.
>
-- 
Michael R. Crusoe: Programmer & Bioinformatician crusoe@ucdavis.edu
The lab for Data Intensive Biology; University of California, Davis
https://impactstory.org/MichaelRCrusoe http://twitter.com/biocrusoe